src/common/constants.ts

```diff
@@ -29,3 +29,14 @@ export const SEARCH_QUERY_VARIABLES = {
     limit: 50,
   },
 };
+
+export const SET_VARIABLES = (filter: any, limit: any, offset: any, order: string) => {
```

Comment:
@DigneshGujarathi may be set defaults to the params so that we can further optimize the rest of the code.